cypress/integration/search.spec.js

```diff
@@ -1,49 +1,49 @@
-/* eslint-disable cypress/no-unnecessary-waiting */
 import {
   CLASSNAME,
-  MAX_KEYWORDS_COUNT,
-  MAX_RESULTS_COUNT,
-} from "../../src/js/constants.js";
+  NUMBER,
+  REGULAR_EXPRESSION,
+} from "../../src/js/constants/index.js";
```

Comment:
index.js에 constants를 전부 모아두는 것 보다, 각각의 요소들이 엄청 커진상황이므로 constants 하위에 각각 파일별로 관리하는게 좋을 것 같아요!

REGULAR_EXPRESSION만 쓰이는 곳이 있다면 다른 파일도 로드를 하는게 수반되어야 하니..

Reply:
각 디렉토리의 index.js는 같은 디렉토리 내의 모듈들을 모두 import 한 후, export만 하는 역할을 수행하고 있습니다.
constants 외의 모듈에서는 constants 디렉토리내의 모든 모듈을 contstants/index.js 로부터 import 하도록 하였습니다.

테스트파일에서는 constants 내 모든 모듈이 import 될 필요가 없으니, 필요한 모듈을 직접 해당 파일로부터 import하는 방식으로 수정하였습니다! 

```suggestion
import CLASSNAME from "../../src/js/constants/classname.js";
import NUMBER from "../../src/js/constants/number.js";
import REGULAR_EXPRESSION from "../../src/js/constants/regularExpression.js";
```

관련 커밋: https://github.com/woowacourse/javascript-youtube-classroom/pull/69/commits/a616f17e67d35d520e9754cc5ca35a734c5a4249